ydb/core/fq/libs/compute/ydb/status_tracker_actor.cpp

```diff
@@ -155,42 +155,46 @@ class TStatusTrackerActor : public TBaseComputeActor<TStatusTrackerActor> {
+            auto publicCounters = GetPublicCounters();
 
-        if (stat.MemoryUsageBytes) {
-            auto& counter = *publicCounters->GetNamedCounter("name", "query.memory_usage_bytes");
-            counter = *stat.MemoryUsageBytes;
-        }
+            if (stat.MemoryUsageBytes) {
+                auto& counter = *publicCounters->GetNamedCounter("name", "query.memory_usage_bytes");
+                counter = *stat.MemoryUsageBytes;
+            }
 
-        if (stat.CpuUsageUs) {
-            auto& counter = *publicCounters->GetNamedCounter("name", "query.cpu_usage_us", true);
-            counter = *stat.CpuUsageUs;
-        }
+            if (stat.CpuUsageUs) {
+                auto& counter = *publicCounters->GetNamedCounter("name", "query.cpu_usage_us", true);
+                counter = *stat.CpuUsageUs;
+            }
 
-        if (stat.InputBytes) {
-            auto& counter = *publicCounters->GetNamedCounter("name", "query.input_bytes", true);
-            counter = *stat.InputBytes;
-        }
+            if (stat.InputBytes) {
+                auto& counter = *publicCounters->GetNamedCounter("name", "query.input_bytes", true);
+                counter = *stat.InputBytes;
+            }
 
-        if (stat.OutputBytes) {
-            auto& counter = *publicCounters->GetNamedCounter("name", "query.output_bytes", true);
-            counter = *stat.OutputBytes;
-        }
+            if (stat.OutputBytes) {
+                auto& counter = *publicCounters->GetNamedCounter("name", "query.output_bytes", true);
+                counter = *stat.OutputBytes;
+            }
 
-        if (stat.SourceInputRecords) {
-            auto& counter = *publicCounters->GetNamedCounter("name", "query.source_input_records", true);
-            counter = *stat.SourceInputRecords;
-        }
+            if (stat.SourceInputRecords) {
+                auto& counter = *publicCounters->GetNamedCounter("name", "query.source_input_records", true);
+                counter = *stat.SourceInputRecords;
+            }
 
-        if (stat.SinkOutputRecords) {
-            auto& counter = *publicCounters->GetNamedCounter("name", "query.sink_output_records", true);
-            counter = *stat.SinkOutputRecords;
-        }
+            if (stat.SinkOutputRecords) {
+                auto& counter = *publicCounters->GetNamedCounter("name", "query.sink_output_records", true);
+                counter = *stat.SinkOutputRecords;
+            }
 
-        if (stat.RunningTasks) {
-            auto& counter = *publicCounters->GetNamedCounter("name", "query.running_tasks");
-            counter = *stat.RunningTasks;
+            if (stat.RunningTasks) {
+                auto& counter = *publicCounters->GetNamedCounter("name", "query.running_tasks");
+                counter = *stat.RunningTasks;
+            }
+        } catch(const NJson::TJsonException& ex) {
+            LOG_E("Error statistics conversion: " << ex.what());
```

Comment:
Do we need to report this information to the user through transient issues?

Reply:
Don't think so. Statistics is not UG, it is created by other part of our software, so bad format is likely to be our bug, user will get no useful information from the error and has no chances to help us or fix the error

Reply:
1. How can user understand that the in progress statistics doesn't work?
2. How can we follow that we have a problem in this place? Do we need any sensors here or not?